src/de/jost_net/JVerein/gui/control/MailControl.java

```diff
@@ -389,11 +389,13 @@ public void handleAction(Object context)
     return b;
   }
 
+  @Override
   public String getBetreffString() throws RemoteException
```

Comment:
Eigentlich könnte man in einem weiteren Clean-Up PR alle Referenzen auf `getBetreff()` refactoren, so dass sie diese Funktion verwenden.